Source/Resources/Content/MSGraph/DeviceManagement/DeviceConfigurations/Security - Password Reset Enabled.json

```diff
@@ -8,7 +8,7 @@
       }
     }
   ],
-  "description": "Enables Azure AD users to reset their passwords from the Windows login screen.",
+  "description": "What is it: Enables Azure AD users to reset their passwords from the Windows login screen. \n Learn More: https://docs.microsoft.com/en-us/azure/active-directory/authentication/howto-sspr-windows",
```

Comment:
**What does this do?**
Enables Azure AD users to reset their passwords from the Windows login screen.

**Why should you use this?**
If you want to allow users to reset their passwords from the Windows login screen.

**What is the end-user impact?**
Users may reset their passwords from the Windows login screen.

**Learn more**
[Enable Azure Active Directory self-service password reset at the Windows sign-in screen](https://docs.microsoft.com/en-us/azure/active-directory/authentication/howto-sspr-windows)